git-sync/version.txt

```diff
@@ -1 +1 @@
-3.6.9
+3.6.9-1
```

Comment:
why is this version being updated is this to check the pipeline

Reply:
@pgvishnuram this was so we could see a build happen, which is needed to trigger the trivy rate limit bug we are investigating.